src/System.Text.Json/tests/Json.cs

```diff
@@ -9,6 +9,132 @@ public class Json
 
         public Array Array { get; set; }
 
+        public Value Value { get; set; }
+
+        public Json this[string index]
+        {
+            get
+            {
+                var json = new Json();
```

Comment:
NIT is short for nitpick...something that can be ignored if you so choose but worth calling out :)